engine/src/main/java/org/terasology/engine/TerasologyEngine.java

```diff
@@ -428,34 +429,34 @@ private void mainLoop() {
 
             Iterator<Float> updateCycles = time.tick();
 
-            PerformanceMonitor.startActivity("Network Update");
-            networkSystem.update();
-            PerformanceMonitor.endActivity();
+            try (Activity ignored = PerformanceMonitor.startActivity("Network Update")) {
```

Comment:
Would an anonymous activity object work, i.e.:

    try (PerformanceMonitor.startActivity("Network Update")) {

 And if it would, would it be sensible to do it that way?